### PR TITLE
Add bulk apis for fdb_entry

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: inc/saifdb.h [hunk 1]
```diff
@@ -409,6 +409,98 @@ typedef void (*sai_fdb_event_notification_fn)(
...
+
+/**
+ * @brief Bulk get attribute on FDB entry
+ *
+ * @param[in] object_count Number of objects to set attribute
+ * @param[in] fdb_entry List of objects to set attribute
+ * @param[in] attr_count List of attr_count. Caller passes the number
+ *    of attribute for each object to get
+ * @param[inout] attr_list List of attributes to set on objects, one attribute per object
+ * @param[in] mode Bulk operation error handling mode
```
Comment (atitjain) [inline inc/saifdb.h]: change "to set" in line 482 and 486 to "to get"

File: inc/saifdb.h [hunk 2]
```diff
@@ -409,6 +409,98 @@ typedef void (*sai_fdb_event_notification_fn)(
...
+ *
+ * @param[in] object_count Number of objects to set attribute
+ * @param[in] fdb_entry List of objects to set attribute
+ * @param[in] attr_list List of attributes to set on objects, one attribute per object
+ * @param[in] mode Bulk operation error handling mode.
+ * @param[out] object_statuses List of status for every object. Caller needs to
+ * allocate the buffer
+ *
+ * @return #SAI_STATUS_SUCCESS on success when all objects are removed or
+ * #SAI_STATUS_FAILURE when any of the objects fails to remove. When there is
```
Comment (atitjain) [inline inc/saifdb.h]: replace remove with set. Same for lines 491-492

File: inc/saifdb.h (hunk 1 above)
Comment (kcudnik) [reply inc/saifdb.h]: done

File: inc/saifdb.h (hunk 2 above)
Comment (kcudnik) [reply inc/saifdb.h]: done

**Pull request merged**